src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+//import MicroscopyMetadataTool from "4dn-microscopy-metadata-tool";
+import { console, object, layout } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { ItemFileAttachment } from './components/ItemFileAttachment';
+import DefaultItemView from './DefaultItemView';
+
+
+export default class MicroscopeConfigurationView extends DefaultItemView {
+
+    getTabViewContents(){
+        const initTabs = [];
+        initTabs.push(MicroMetaTabView.getTabObject(this.props));
+        return initTabs.concat(this.getCommonTabs()); // Add remainder of common tabs (Details, Attribution)
+    }
+
+}
+
+
+
+
+function onLoadSchema(complete) {
+    // Maybe some UI to select something...
+    // Not all browsers have `window.fetch`, used for demoing purposes.
+    // Also, window.fetch requires HTTP so we getting this from GitHub... lol
+    window
+        .fetch(
+            "https://raw.githubusercontent.com/WU-BIMAC/4DNMetadataSchemaXSD2JSONConverter/master/fullSchema.json"
+        )
+        .then(function(resp) {
+            console.log(resp);
+            return resp.text();
+        })
+        .then(function(respText) {
+            var schema = JSON.parse(respText);
+            complete(schema);
+        });
+}
+
+function onLoadMicroscopes(complete) {
+    const microscopesDB = {};
+    complete(microscopesDB);
+}
+
+function onSaveMicroscope(microscope, complete) {
+    // Do some stuff... show pane for people to browse/select schema.. etc.
+    setTimeout(function() {
+        console.log(microscope);
+        complete(microscope.Name);
+    });
+}
+
+const imagesPath = "https://raw.githubusercontent.com/WU-BIMAC/4DNMicroscopyMetadataToolReact/master/public/assets/";
```

Comment:
Can probably leave this hardcoded. Maybe we can have `4dn-microscopy-metadata-tool` export out a "cdnImagesPath" that would be most up-to-date. Idk.